src/CameraControls.ts

```diff
@@ -553,8 +553,10 @@ export class CameraControls extends EventDispatcher {
 			const dragging = ( event: Event ): void => {
 
 				if ( ! this._enabled ) return;
-
-				event.preventDefault();
+				
+				if (event.cancelable) {
+					event.preventDefault();
+				}
```

Comment:
Would you mind adapting the lint as well?
```suggestion
				
				if ( event.cancelable ) event.preventDefault();
```